github/AuthenticatedUser.py

```diff
@@ -177,7 +177,7 @@ def _initAttributes(self) -> None:
         self._two_factor_authentication: Attribute[bool] = NotSet
 
     def __repr__(self) -> str:
-        return self.get__repr__({"login": self._login.value})
+        return self.get__repr__({"login": self._login.value or "<UNKNOWN>"})
```

Comment:
I think `self._login.value or "<UNKNOWN>"` does not work as you would expect.
```suggestion
        return self.get__repr__({"login": self._login.value if is_defined(self._login.value) else "<UNKNOWN>"})
```